code/modules/mob/living/carbon/emote.dm

```diff
@@ -53,6 +53,38 @@
 		return
 	var/mob/living/carbon/human/human_user = user
 	QDEL_IN(human_user.give_emote_overlay(/datum/bodypart_overlay/simple/emote/cry), 12.8 SECONDS)
+	var/area/station/medical/psychology/weekly_session = astype(get_area(human_user))
+	if(isnull(weekly_session))
+		return
```

Comment:
for me, as a gift (reading it does not require understanding astype, and it's less code)
```suggestion
	var/area/home = get_area(human_user)
	if(istype(home, /area/station/medical/psychology))
		return
```